pontoon/sync/tests/formats/test_xliff.py

```diff
@@ -103,15 +102,49 @@ def test_parse_invalid_translation(self):
                 )
             )
 
-    def generate_xliff(self, body, locale_code="en"):
-        return XLIFF_TEMPLATE.format(body=body, locale_code=locale_code)
+    def generate_xliff(self, body, attr=""):
+        return XLIFF_TEMPLATE.format(body=body, attr=attr)
 
     def test_save_basic(self):
         """
         Test saving changes to an entity with a single translation.
```

Comment:
Adjust the comment here to highlight that the `target-language` attribute exists?

Not sure, should this test actually test that we're not modifying the in-file locale code? That is, not use `self.locale.code`, at least not w/out a suffix?